Realm/RLMUser.h

```diff
@@ -66,29 +66,26 @@ NS_ASSUME_NONNULL_BEGIN
  The unique MongoDB Realm string identifying this user.
  Note this is different from an identitiy: A user may have multiple identities but has a single indentifier. See RLMUserIdentity.
  */
-@property (nullable, nonatomic, readonly) NSString *identifier NS_SWIFT_NAME(id);
+@property (nonatomic, readonly) NSString *identifier NS_SWIFT_NAME(id);
 
-/**
- Returns an array of identities currently linked to a user.
-*/
-- (NSArray<RLMUserIdentity *> *)identities;
+/// Returns an array of identities currently linked to a user.
+@property (nonatomic, strong, readonly) NSArray<RLMUserIdentity *> *identities;
 
 /**
  The user's refresh token used to access the Realm Applcation.
 
  This is required to make HTTP requests to the Realm App's REST API
  for functionality not exposed natively. It should be treated as sensitive data.
  */
-@property (nullable, nonatomic, readonly) NSString *refreshToken;
-
+@property (nonatomic, readonly) NSString *refreshToken;
 
 /**
  The user's refresh token used to access the Realm Application.
 
  This is required to make HTTP requests to MongoDB Realm's REST API
  for functionality not exposed natively. It should be treated as sensitive data.
  */
-@property (nullable, nonatomic, readonly) NSString *accessToken;
+@property (nonatomic, readonly) NSString *accessToken;
```

Comment:
Access token and refresh token actually can be nil, due to logged in/logged out state.